app/src/test/java/org/simple/clinic/registration/pin/RegistrationPinScreenControllerTest.kt

```diff
@@ -49,6 +49,9 @@ class RegistrationPinScreenControllerTest {
     uiEvents.onNext(RegistrationPinTextChanged("1234"))
 
     verify(userSession).saveOngoingRegistrationEntry(ongoingRegistrationEntry.withPin("1234"))
```

Comment:
Should we also add `verifyNoMoreInteractions` for `UserSession`?

Reply:
We can, I'll do this.

Reply:
Done